app/domain/case-building.ts

```diff
@@ -0,0 +1,3 @@
+interface CaseBuilding {
```

Comment:
Can we move this to types folder to have all the interfaces and types in one place?